src/core.spec.js

```diff
@@ -0,0 +1,12 @@
+/* global beforeEach, describe, expect, it */
```

Comment:
Let's keep the tests in a `/tests` folder. I like that they are named `*.spec.js` though.